### PR TITLE
Prepares for release of v2021.1 (Engine)

### PR DESCRIPTION
**Pull request by matsduf**
Updates:
* Engine.pm with new version (v4.2.0)
* Changes
* Makefile.PL (updated dependency)

### CODE REVIEW

**Comment by matsduf** [comment]
Fails until https://github.com/zonemaster/zonemaster-ldns/pull/121 has been merged.

File: Changes [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+ - Updates profile to reduce repeated messages on no response (#923)
+ - Adds test case implementation of DNSSEC17 (#906, #898)
+ - Adds test case implementation of DNSSEC16 (#900, #897)
+ - Adds test case implementation of DNSSEC15 (#896, #894)
+ - Adds test case implementation of Basci04 (#892, #763)
+ - Adds Finnish translation (PO file) (#880, #879)
+
+ [Fixes]
+ - Updates translations (#921, #917, #916, #914)
+ - Updates instructions (#918, #919, #889)
```
Comment (mattias-p) [inline Changes:15]: I have another update to the CentOS installation instruction. It's not necessary but I think it's nice. I realized that gcc is the only thing we need from the "Development tools" group. So it's a pretty significant reduction in dependency footprint - at least as far as the installation instruction goes. And I've made a couple of other tweaks as well. I haven't made the PR yet. I'll make it right now.

Comment (matsduf) [reply Changes:15]: I will add that and assume that you will merge it for this release.

**Review by mattias-p — DISMISSED** [review]

**Comment by matsduf** [comment]
@pnax and @mattias-p, this should be the last update. Please rereview.

**Review by pnax — APPROVED** [review]
> - Adds Finnish translation (PO file) (#880, #879)

Ok for me even though #911 has not been done yet.

**Comment by matsduf** [comment]
> > * Adds Finnish translation (PO file) (#880, #879)
> 
> Ok for me even though #911 has not been done yet.

The bulk part of translation to Finnish has been completed. The need for update is mostly due to the addition of Basic04 and DNSSEC15-17. The Finnish update can come in a patch release in a couple of weeks.

**Review by mattias-p — APPROVED** [review]

**Pull request merged**